info.yml

```diff
@@ -3,6 +3,25 @@
 # expected fields in the configuration file for this engine
 configuration:
 
+    debug_logging:
+        type: bool
+        description: Controls whether debug messages should be emitted to the logger
+        default_value: false
+
+    menu_favourites:
+        type: list
+        description: "Controls the favourites section on the main menu. This is a list
+                     and each menu item is a dictionary with keys app_instance and name.
+                     The app_instance parameter connects this entry to a particular
+                     app instance defined in the environment configuration file. The name
+                     is a menu name to make a favourite."
```

Comment:
Nice, I know shotgun doesn't fully utilize yaml to it's fullest, but you can also use the `: > ` format

Check out the **demo** in https://yaml-multiline.info/